docs/running-tests.rst

```diff
@@ -113,6 +113,13 @@ It is recommended (but not necessary) to use vagrant from
 ``quay.io/sssd/vagrant:latest`` container to avoid issues with vagrant plugin
 installation.
 
+.. note::
+  
+    If it is the first time you are running a virtual machine using libvirt,
+    you need to grant the user permissions (ACL) to access the libvirt daemon. You
+    can do this by adding the user to the ``libvirt`` or ``qemu`` user.
+    https://github.com/jedi4ever/veewee/issues/996#issuecomment-375931044
```

Comment:
Isn't there any official documentation that we can link to?

Reply:
It doesn't explicitly say it.  

```
Warning

Before you start using vagrant-libvirt, please make sure your Libvirt and QEMU installation is working correctly and you are able to create QEMU or KVM type virtual machines with virsh or virt-manager.
```
https://vagrant-libvirt.github.io/vagrant-libvirt/

Reply:
I'm sure there used to be some official guide, since you had to also change polkit rule. But maybe it was changed... I'm fine with the current link